vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/exporter/ZephyrExporter.java

```diff
@@ -0,0 +1,167 @@
+import org.vividus.zephyr.model.ZephyrExecution;
+
+public class ZephyrExporter
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZephyrExporter.class);
+
+    private JiraFacade jiraFacade;
+    private IZephyrFacade zephyrFacade;
+    private ZephyrExporterProperties zephyrExporterProperties;
+
+    public void exportResults() throws IOException
+    {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        objectMapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
+        objectMapper.registerModule(new SimpleModule().addDeserializer(TestCase.class, new TestCaseDeserializer()));
+
+        List<TestCase> testCases = parseJsonResultsFile(getJsonResultsFiles(), objectMapper);
+        LOGGER.info("Test cases: {}", testCases);
+        Map<String, Set<TestCaseStatus>> testCasesMap = testCases.stream()
+                .collect(Collectors.groupingBy(TestCase::getKey,
+                        Collectors.mapping(TestCase::getStatus, Collectors.toSet())));
+
+        List<TestCase> testCasesForImporting = new ArrayList<>();
+        for (Map.Entry<String, Set<TestCaseStatus>> entry : testCasesMap.entrySet())
+        {
+            TreeSet<TestCaseStatus> statuses = new TreeSet<>(entry.getValue());
+            testCasesForImporting.add(new TestCase(entry.getKey(), statuses.first()));
+        }
+        LOGGER.info("Test cases for exporting to JIRA: {}", testCasesForImporting);
+        ZephyrConfiguration configuration = zephyrFacade.prepareConfiguration();
+        for (TestCase testCase : testCasesForImporting)
+        {
+            createNewTestExecution(testCase, configuration, objectMapper);
+        }
+    }
+
+    private List<File> getJsonResultsFiles() throws IOException
+    {
+        List<File> jsonFiles = Files.walk(zephyrExporterProperties.getSourceDirectory())
+                .map(Path::toFile)
+                .filter(File::isFile)
+                .filter(f -> f.getPath().contains("test-cases"))
+                .collect(Collectors.toList());
+
+        notEmpty(jsonFiles, "Folder '%s' does not contain needed json files",
+                zephyrExporterProperties.getSourceDirectory());
+        LOGGER.info("Json files: {}", jsonFiles);
+        return jsonFiles;
+    }
+
+    private List<TestCase> parseJsonResultsFile(List<File> jsonFiles, ObjectMapper objectMapper)
+    {
+        List<TestCase> testCases = new ArrayList<>();
+        for (File jsonFile : jsonFiles)
+        {
+            try
+            {
+                TestCase testCase = objectMapper.readValue(jsonFile, TestCase.class);
+                if (testCase.getKey() == null)
+                {
+                    continue;
+                }
+                if (testCase.getKey().contains(" "))
+                {
+                    String[] keys = testCase.getKey().split("\\s+");
```

Comment:
we use semicolon as separator by default

Reply:
for meta also ?

I thought it should be looks like `@testCaseId test-1 test-2`

Reply:
yep, for meta: https://github.com/vividus-framework/vividus/blob/master/vividus-bdd-engine/src/main/java/org/vividus/bdd/model/MetaWrapper.java#L35